kml/serdes_gpx.cpp

```diff
@@ -129,6 +130,14 @@ std::string_view const & GpxParser::GetTagFromEnd(size_t n) const
   return m_tags[m_tags.size() - n - 1];
 }
 
+void GpxParser::ParseColor(std::string const & value)
+{
+  auto const fromHex = FromHex(value);
+  if (fromHex.size() != 3)
+    return;
+  m_color = ToRGBA(fromHex[0], fromHex[1], fromHex[2], '\377');
```

Comment:
Octal system is not understandable by many. Why not use just a plain 255 number?